go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/DataDog/zstd v1.4.1 // indirect
 	github.com/Sereal/Sereal v0.0.0-20190618215532-0b8ac451a863 // indirect
+	github.com/armon/go-socks5 v0.0.0-20160902184237-e75332964ef5
```

Comment:
This library was last time updated 3 years ago. It may still be stable and good quality but maybe you could take a look at golang/x/net is suitable https://github.com/golang/net/blob/master/proxy/socks5.go?